charts/datadog-operator/values.yaml

```diff
@@ -176,3 +176,13 @@ volumeMounts: []
 #   - name: <VOLUME_NAME>
 #     mountPath: <CONTAINER_PATH>
 #     readOnly: true
+
+# livenessProbe -- Add default livenessProbe settings
+livenessProbe:
+  periodSeconds: 10
+  httpGet:
+    path: /healthz/
+    port: 8081
+
+# healthPort -- Port number to use in the Operator for the healthz endpoint
+healthPort: 8081
```

Comment:
We can't parametrize these two unfortunately as they are hard-coded in Operator.

https://github.com/DataDog/datadog-operator/blob/3c38ef29ed6f992459e5a2df4af07a883a73258b/cmd/main.go#L238

https://github.com/DataDog/datadog-operator/blob/3c38ef29ed6f992459e5a2df4af07a883a73258b/pkg/kubernetes/rbac/const.go#L91

We can leave these hard-coded in the template but add [settings](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) which can change behavior of the liveness probe. For example, [this](https://github.com/DataDog/helm-charts/blob/451b67348d51c0375613353f359042dea14bc66c/charts/datadog/values.yaml#L1276-L1281) is what Agent chart does, we can support same config in Operator.

Reply:
Would it be okay to keep healthPort (line 187-188)? Like how it is configured in the Agent chart